test/diamond-pattern/diamond.helper.js

```diff
@@ -19,77 +19,81 @@
  */
 
 let DiamondHelper = {
-  FacetCutAction: {Add: 0, Replace: 1, Remove: 2},
+  FacetCutAction: { Add: 0, Replace: 1, Remove: 2 },
 
   getSelectors: function (contract) {
-    const selectors = Object.keys(contract.interface.functions).reduce((acc, val) => {
-      if (val !== 'init(bytes)') {
-        acc.push(contract.interface.getSighash(val));
-      }
-      return acc;
-    }, []);
+    const selectors = Object.keys(contract.interface.functions).reduce(
+      (acc, val) => {
+        if (val !== 'init(bytes)') {
+          acc.push(contract.interface.getSighash(val))
```

Comment:
I guess prettier really doesn't believe in end-of-statement/end-of-line `;`.  Is _not_ having them the prevailing standard in the JS community?